include/pmacc/cuplaHelper/ValidateCall.hpp

```diff
@@ -24,11 +24,10 @@
 #pragma once
 
 #include <cupla.hpp>
+
 #include <iostream>
 #include <stdexcept>
 
-namespace pmacc
-{
```

Comment:
Note: this is unrelated, just was noticed during the development. Since there are only macros in this file, no sense to open the namespace.